money_helper.gemspec

```diff
@@ -15,7 +15,7 @@ Gem::Specification.new do |s|
   s.require_paths = ['lib']
   s.homepage = 'https://github.com/artsy/money_helper'
   s.add_dependency('activesupport')
-  s.add_dependency('money', '~> 6.5')
+  s.add_dependency('money', '6.16.0')
```

Comment:
The `money` gem updates its [currency data](https://github.com/RubyMoney/money/commits/main/config/currency_iso.json) fairly frequently, including subunit to unit conversion, symbols, disambiguation. We had an issue previously that 2 clients were on different version of the gem and the cents to dollars conversion was messed up due to the subunit to unit ratio mismatch. Maybe it makes sense to require a specific version for clients?